custom_command_logic/pso2/itemname.js

```diff
@@ -1,5 +1,8 @@
 import {BaseCommand} from "../util/BaseCommand";
 import {Message} from "discord.js";
+import  {DMChannel} from "discord.js";
```

Comment:
Merge the 3 imported classes from one file into one like so:
`{Message, DMChannel, TextChannel}`
instead of using separate import statements